scripts/ensure-stale-workflow-is-in-all-repos.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+set -o pipefail
+
+root="$(realpath "$(dirname "$0")/..")"
+
+repository_file="$(jq '.' "$root/github/ipfs/repository_file.json")"
+
+while read repository; do
+  echo "Checking if $repository has stale.yml already"
+  has_stale="$(jq '.[$repository] | .[".github/workflows/stale.yml"] | . != null' --arg repository "$repository" <<< "$repository_file")"
+  if [[ "$has_stale" == 'true' ]]; then
+    echo "stale.yml already exists in $repository, skipping"
+  else
+    echo "Checking if $repository has either needs/author-input or hints/needs-author-input label"
+    if gh api repos/ipfs/$repository/labels/need/author-input > /dev/null 2>&1 || gh api repos/ipfs/$repository/labels/hint/needs-author-input > /dev/null 2>&1; then
```

Comment:
Could we clean this up so there is only one label?
- add `need/author-input` if missing
- remove `hint/needs-author-input`  and re-assign issues to  `need/author-input`

:pray:

Reply:
Sure, no problem.

Reply:
Ad.2. turns out `hint/needs-author-input` does not exist on any repositories and I cannot find any issues labelled with it https://github.com/search?q=org%3Aipfs+label%3Ahint%2Fneeds-author-input

Reply:
Ad.1. I did it in this commit https://github.com/ipfs/github-mgmt/pull/4/commits/6768fbae84651c7ec0a804b0e37cb13a71240a47. I also included the script in which I did it there.